### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by r-ahmed2022**
In this PR I have completed the following tasks:- 
-   Created a table named owners with the following columns.
-  Created a table named species.
-  Modified animals table so that  is set as autoincremented PRIMARY KEY
      - Removed column species
      - Added column species_id which is a foreign key referencing species table
     - Added column owner_id which is a foreign key referencing the owners table
-  added all these to the schema.sql file.
- Inserted the  data into the owners table:
- Inserted the data into the species table:
- Modified your inserted animals so it includes the species_id value:
- Modified your inserted animals to include owner information (owner_id):
- Added these queries to the data.sql file.
- Wrote queries (using JOIN) to filter 
- Added sql queries to the queries.sql file.
- Took a screenshot of the results of the queries, and link below 
    https://docs.google.com/document/d/1TEGf_bvV7Xs8DAgQV-65majdvJSYF3MjTYIx11824s8/edit?usp=sharing

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -10,4 +10,62 @@ insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weigh
...
+(5, 'Charmander', '2020-02-08', 0, false, -11)
+(6, 'Plantmon', '2021-11-15', 2, true, -5.7)
+(7, 'Squirtle', '1993-04-02', 3, true, -12.13) , 
+(8, 'Angemon', '2005-06-12', 1, true, -45) ,
+(9, 'Boarmon', '2005--07-07', 7, true, 20.4) ,
+(10, 'Blossom', '1998-10-13', 3, true, 17) ,
+(11, 'Ditto', '2022-05-14', 4, true, 22)
+
+
+insert into owners (full_name, age)
```
Comment (1GORDON) [inline data.sql:26]: Great job @r-ahmed2022 🤝

- [ ] As stated in the project requirements, kindly add all the data to the `owners` table.
melody

File: schema.sql [hunk 2]
```diff
@@ -7,5 +7,68 @@ CREATE TABLE animals(
...
+);
+
+select * from owners
+
+create table species(
+	id serial,
+	full_name varchar(40),
+	age int,
+	primary key(id)
+);
```
Comment (1GORDON) [inline schema.sql]: Great job @r-ahmed2022 🤝

- [ ] As stated in the project requirements, kindly remove the `age` column from the `species` table.
species

- [ ] [Optional] You may as well update `full-name` to `name`.

File: queries.sql [hunk 3]
```diff
@@ -15,3 +15,50 @@ select * from animals where neutered = true
 select * from animals where name not like '%Gabumon'
 
 select * from animals where weight_kg between  10.4 and 17.3
+
```
Comment (1GORDON) [inline queries.sql:18]: Great job @r-ahmed2022 🤝

- [ ] Kindly add all queries for this milestone to the `queries.sql` file.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @r-ahmed2022 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (r-ahmed2022) [reply schema.sql]: @1GORDON ,  In my opinion, there is no age column in the species table,  kindly check

File: queries.sql (hunk 3 above)
Comment (r-ahmed2022) [reply queries.sql:18]: done, as advised

File: data.sql (hunk 1 above)
Comment (r-ahmed2022) [reply data.sql:26]: done, as advised

**Review by uwadonat — APPROVED** [review]
## Hi @r-ahmed2022,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good commit history ✔️
- Correct workflow ✔️
- Well organized and clean project ✔️


thanks-awesome

# Status: Approved ✔️✔️✔️
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**